lib/blockscore/base.rb

```diff
@@ -53,22 +52,22 @@ def self.resource
       @resource ||= Util.to_underscore(to_s.split('::').last)
     end
 
-    def self.api_url
-      'https://api.blockscore.com/'
-    end
-
     def self.endpoint
       fail NotImplementedError, ABSTRACT_WARNING if equal?(Base)
 
-      "#{api_url}#{Util.to_plural(resource)}"
+      API_URL + Util.to_plural(resource)
     end
 
     def persisted?
-      !id.nil? && !attributes[:deleted]
+      !id.nil? && !attributes.fetch(:deleted, false)
```

Comment:
Should this than better be written as:

```ruby
def persisted?
  !(id.nil? || deleted?)
end
```

Since it is equivalent under demorgans law and has less AST nodes, at equivalent maximal depth.